t/lib/t/MusicBrainz/Script/Utils.pm

```diff
@@ -0,0 +1,63 @@
+package t::MusicBrainz::Script::Utils;
+
+use strict;
+use warnings;
+
+use File::Spec;
+use File::Temp qw( tempdir );
+use Test::More;
+use Test::Routine;
+
+use MusicBrainz::Script::Utils qw( find_mbdump_file );
+
+test 'find_mbdump_file works as expected' => sub {
+    my $test = shift;
+
+    my $dir1 = tempdir('find_mbdump_file-XXXXXXXX', DIR => '/tmp', CLEANUP => 1);
+    my $dir2 = File::Spec->catdir($dir1, 'mbdump');
+    my $dir3 = File::Spec->catdir($dir1, 'recording');
+
+    system('mkdir', $dir2, $dir3);
+
+    my $f1_path = File::Spec->catfile($dir1, 'artist');
+    my $f2_path = File::Spec->catfile($dir2, 'recording');
+    my $f3_path = File::Spec->catfile($dir2, 'artist');
+
+    system('touch', $f1_path, $f2_path, $f3_path);
+
+    my ($result, @result);
+
+    $result = find_mbdump_file('dne', $f1_path);
+    is($result, undef,
+       'undef is returned for non-existent file in scalar context');
+
+    @result = find_mbdump_file('dne', $f1_path);
+    is(scalar @result, 0,
+       'empty list is returned for non-existent file in list context');
+
+    $result = find_mbdump_file('recording', $dir3);
+    is($result, undef,
+       'undef is returned for file matching directory name ' .
+       'in scalar context');
+
+    $result = find_mbdump_file('artist', $f1_path);
+    is($result, $f1_path, 'scalar file is found by direct path');
+
+    $result = find_mbdump_file('artist', $dir1);
+    is($result, $f1_path, 'scalar file is found in directory');
```

Comment:
Is there a reason you prefer lowercase for these labels? I generally start them with uppercase because they look like full sentences to me the way we print them, but obviously this is mega minor anyway :)

Reply:
I guess because they read more like fragments than full sentences to me. :) And some of them start with `undef` or variable names which would be weird to capitalize...

I don't have a particular preference though.